src/style.css

```diff
@@ -0,0 +1,91 @@
+@font-face {
+  font-family: "Avenir-Book";
+  src: url(fonts/Avenir-Book.ttf);
```

Comment:
please follow the [checklist](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md)
<img width="1034" alt="Screenshot 2023-01-27 at 11 05 48 AM" src="https://user-images.githubusercontent.com/60295799/215048681-476c059a-7885-46d0-9d69-0d2f54633249.png">